src/server/json_family_test.cc

```diff
@@ -110,7 +110,9 @@ TEST_F(JsonFamilyTest, SetGetFromPhonebook) {
   EXPECT_EQ(resp, R"(["New York","NY","21 2nd Street","10021-3100"])");
 
   resp = Run({"JSON.GET", "json", "$.firstName", "$.age", "$.lastName"});
-  EXPECT_EQ(resp, R"({"$.age":[27],"$.firstName":["John"],"$.lastName":["Smith"]})");
+  EXPECT_EQ(
```

Comment:
could you please add few tests covering those cases?

Reply:
I reverted the change that affected the "JSON.GET" command when formatting args are not supplied with it.
Can you elaborate on which tests are currently missing?

Reply:
I added a few tests based on the previous JSON.GET cases that cover the feature functionality.